### PR TITLE
Update CONTRIBUTING.md for new contributors

### PR DESCRIPTION
**Pull request by jenweber**
I looked over the README and CONTRIBUTING.md and these are the only changes I have for this project. I'll look over the retext plugins next.

I added commands for running from the command line locally and running the tests, plus a more detailed description of which tools do what. I also softened the statement at the top. I think maybe if a stronger disclaimer is needed, it should not be in the "contributing" section. Let me know what you think. I also think it's ok as it was, so I am happy to put it back if you'd like.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ a moment to read the following guidelines.  And thanks for contributing to
 **alex**!  👏👌✨
 
 Before anything else: people involved with this project often do so for fun,
-next to their day job: you are not entitled to free customer service.
+as volunteers, next to their day job. Please be considerate in requests for
```
Comment (wooorm) [inline contributing.md]: Could you use two spaces here between the sentences?

File: contributing.md [hunk 2]
```diff
@@ -8,7 +8,11 @@ a moment to read the following guidelines.  And thanks for contributing to
...
+features and changes, and be patient regarding response times. Maintainers
```
Comment (wooorm) [inline contributing.md]: Same with spaces here!

File: contributing.md [hunk 3]
```diff
@@ -24,11 +28,33 @@ next to their day job: you are not entitled to free customer service.
...
 ## Ecosystem
 
-This project, alex, is a small wrapper around the [unified][] ecosystem.
+This project, `alex`, is a small wrapper around the [unified][] ecosystem.
 It mainly uses [`retext`][retext] to process natural language, with the
 [`retext-equality`][equality] and [`retext-profanities`][profanities] plugins.
 For markdown, [`remark`][remark] is used.
+
 Try and pick the right place to contribute to so we can help you faster.
+`alex` handles the user's command line experience, while the `retext`
```
Comment (wooorm) [inline contributing.md]: Could you use a curly apostrophe in `user's`? So `user’s`.

File: contributing.md [hunk 4]
```diff
@@ -24,11 +28,33 @@ next to their day job: you are not entitled to free customer service.
...
+plugins determine the rules and recommendations.
+
+## Trying out your local changes
+
+To see how your local changes affect `alex`, you can use
+[npm link](https://docs.npmjs.com/cli/link)
+
+```
```
Comment (wooorm) [inline contributing.md]: Could you use `sh` for the language flag of the code block?

File: contributing.md [hunk 5]
```diff
@@ -24,11 +28,33 @@ next to their day job: you are not entitled to free customer service.
...
+git clone <your fork url>
+cd alex
+npm install
+npm link
+alex some-file.md
+```
+
+## Running tests
+
+```
```
Comment (wooorm) [inline contributing.md]: Same flag here!

**Comment by jenweber** [comment]
@wooorm done!

**Comment by wooorm** [comment]
Perfect, squee! 🤗

**Pull request merged**